From 14e3ed204863090f0cdc0cf0c4a4697e6af54109 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kai=20K=C3=B6hne?= <kai.koehne@qt.io>
Date: Thu, 30 Oct 2025 12:06:40 +0100
Subject: [PATCH 234/239] Doc: Fix link to QJSEngine::uiLanguage

uiLanguage is a property of QJSEngine, not QQmlEngine.

Pick-to: 6.8 6.5
Change-Id: Id865f969119d8a5aef2e46d1b673db97da6e40f9
Reviewed-by: Ulf Hermann <ulf.hermann@qt.io>
(cherry picked from commit c87d220405ce9010a58bb91ab44bb859ff78577a)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qml/qml/qqmlengine.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qml/qml/qqmlengine.cpp b/src/qml/qml/qqmlengine.cpp
index 338074969b..c6ed138ae2 100644
--- a/src/qml/qml/qqmlengine.cpp
+++ b/src/qml/qml/qqmlengine.cpp
@@ -988,7 +988,7 @@ void QQmlEngine::captureProperty(QObject *object, const QMetaProperty &property)
   \since 5.15
 
   The uiLanguage holds the name of the language to be used for user interface
-  string translations. It is exposed in C++ as QQmlEngine::uiLanguage property.
+  string translations. It is exposed in C++ as \l QJSEngine::uiLanguage property.
 
   You can set the value freely and use it in bindings. It is recommended to set it
   after installing translators in your application. By convention, an empty string
-- 
2.51.2

